cmd/sync-tags/main.go

```diff
@@ -182,11 +184,29 @@ func main() {
 	// create or update tags from kTagCommits as local tags with the given prefix
 	createdTags := []string{}
 	for name, kh := range kTagCommits {
+		// temporarily publish only alpha tags
+		if !strings.Contains(name, "alpha") {
```

Comment:
Was this present when you ran in the prow test org? If so, how did the rc.2 tag get published?

Reply:
Not present, and I manually deleted the rc2, beta2 and alpha0 tags in published repos to trigger runs for those. If not, sync-tags wouldn't sync any tags (it syncs only newer tags).

Reply:
I would drop this, and just do dry runs against the test org

Reply:
> I would drop this, and just do dry runs against the test org

If we drop this and merge this PR before Monday, `kubernetes-1.17.0` and `v0.17.0` would be automatically published once `v1.17.0` is tagged in k/k... I don't think that would give us enough time to do a run of `v1.17.0` on a test org.

Reply:
would it make sense to:
1. separately merge a commit that skips syncing tags
2. let k8s cut v1.17.0
3. make this branch re-enable syncing tags, and do test runs against a test org
4. merge this and re-enable tag sync once ready?

that would allow us to get this in for 1.17.0 without rushing

Reply:
Sounds good. :+1: 

Will make the changes in a bit.

Reply:
Dropped this change and created https://github.com/kubernetes/kubernetes/pull/86008 to disable syncing tags.